packages/angular/src/utils/mfe/with-module-federation.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  SharedLibraryConfig,
+  sharePackages,
+  shareWorkspaceLibraries,
+} from './mfe-webpack';
+import { createProjectGraphAsync } from '@nrwl/workspace/src/core/project-graph';
+import { readWorkspaceJson } from '@nrwl/workspace';
+import { joinPathFragments } from '@nrwl/devkit';
+import ModuleFederationPlugin = require('webpack/lib/container/ModuleFederationPlugin');
+
+export type MFERemotes = string[] | [remoteName: string, remoteUrl: string][];
+
+export interface MFEConfig {
+  name: string;
+  remotes?: MFERemotes;
+  exposes?: Record<string, string>;
+  shared?: (
+    libraryName: string,
+    library: SharedLibraryConfig
+  ) => SharedLibraryConfig | false;
+}
+
+async function getDependentPackagesForProject(name: string) {
+  const projectGraph = await createProjectGraphAsync();
```

Comment:
Can use `readCachedProjectGraph` here instead, which will always be present  since the build is invoked through Nx. It'll make the builds a bit faster.

Reply:
Wasn't aware of that, awesome